toml/src/main/kotlin/org/rust/toml/inspections/CrateNotFoundInspection.kt

```diff
@@ -55,7 +55,10 @@ class CrateNotFoundInspection : TomlLocalInspectionToolBase() {
             if (property in dependency.properties) return
         }
 
-        if (CratesLocalIndexService.getInstance().getCrate(dependency.crateName) == null) {
+        val cratesService = CratesLocalIndexService.getInstance()
+        if (cratesService.isUpdating) return
```

Comment:
Discussed in person that it might be better to throw an exception in case of the service update